src/components/pageTemplates/NodeTprUnitPage.tsx

```diff
@@ -109,42 +110,50 @@ function NodeTprUnitPage({
         {opening_hours && (
           <Block
             title={t('unit.open_hours')}
-            icon='clock'
+            icon="clock"
             content={[opening_hours[0].value]}
           />
         )}
 
+        {email && service_map_embed === '53341' && (
+          <Block
+            title={t('unit.email')}
+            icon="envelope"
+            content={[`${email}`]}
```

Comment:
This should probably be a `mailto:` link, at least that's how other emails are handled.

This seemed to work, but we should check if this is actually valid react and if there's a better element / component to use: 
```
<Block
  title={t('unit.email')}
  icon="envelope"
  content={[`<a href="mailto:${email}">${email}</a>`]}
/>
```